packages/react/src/components/OverflowMenu/OverflowMenu.story.jsx

```diff
@@ -41,6 +43,21 @@ const props = {
   }),
 };
 
+OverflowMenu.displayName = 'OverflowMenu';
+
+export default {
+  title: 'OverflowMenu',
```

Comment:
For these extended OverflowMenu and Tooltip the stories should be in the IoT section.

Reply:
@davidicus Moved these into IoT